pkg/report/types/types.go

```diff
@@ -34,6 +35,7 @@ type InstanceData struct {
 	ResourceStates  appstatetypes.ResourceStates `json:"resource_states"`
 	K8sVersion      string                       `json:"k8s_version"`
 	K8sDistribution string                       `json:"k8s_distribution"`
+	Tags            tagstypes.InstanceTagData    `json:"tags"`
```

Comment:
I think we should factor these into `SendAirgapInstanceData` as well. This way the tags are saved to the instance report secret and can later be picked up by airgap custom metrics. To test this, you can install the SDK with the `isAirgap: true` value and check the secret to make sure it's saved. Data in that secret is gzipped and base64 encoded, so you can use something like this to check it:
```
kubectl get secret replicated-instance-report -n <namespace> -o jsonpath='{.data.report}' | base64 -d | base64 -d | gzip -d | jq
```

Reply:
@cbodonnell Where do you set the `isAirgrap: true`?

Reply:
@FourSigma I would probably test this out with a standalone Helm install of the SDK.

Login to the Replicated registry with a customer license email/id and install the SDK standalone in the environment with a dev image:
```
helm install replicated oci://registry.replicated.com/library/replicated --version 1.0.0-beta.16 -n replicated --create-namespace --set isAirgap=true --set images.replicated-sdk=<your-ttl.sh-image>
```

Reply:
Thanks for the tips! I didn't not realize there was a Helm parameter to change the replicated image!  I manually modified the Deployment spec to get it working locally.